packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -507,6 +532,10 @@ enum BillingChoiceMode {
   /// Billing through app provided flow.
   @JsonValue(1)
   alternativeBillingOnly,
+
+  /// Users can choose play billing or alternative billing.
```

Comment:
Nit: Play

Reply:
Done and fixed some other locations in this file.